torchtune/models/llama3/_tokenizer.py

```diff
@@ -166,11 +166,34 @@ def decode(
         Returns:
             str: The decoded string.
         """
-        return self.tt_model.decode(
-            token_ids,
-            truncate_at_eos=truncate_at_eos,
-            skip_special_tokens=skip_special_tokens,
-        )
+        if skip_special_tokens:
```

Comment:
will this test pass? Probably not well formulated, but i want to see if after every encode/decode we are adding \n\n

`text = decode(encode(text), skip_special_tokens=True)`